ecosystem/ffmpeg_plugin/kahawai_common.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Kahawai common struct and functions
+ * Copyright (c) 2023 Intel
+ *
+ * FFmpeg is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * FFmpeg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with FFmpeg; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include <mtl/st_convert_api.h>
+#include <mtl/st_pipeline_api.h>
+
+#include "libavutil/rational.h"
+
+#include "kahawai_common.h"
+
+const KahawaiFpsDecs fps_table[] = {
+    {ST_FPS_P50, 5000 - 100, 5000 + 100},
+    {ST_FPS_P29_97, 2997 - 100, 2997 + 100},    {ST_FPS_P25, 2500 - 100, 2500 + 100},
+    {ST_FPS_P60, 6000 - 100, 6000 + 100},       {ST_FPS_P30, 3000 - 100, 3000 + 100},
+    {ST_FPS_P24, 2400 - 100, 2400 + 100},       {ST_FPS_P23_98, 2398 - 100, 2398 + 100},
+    {ST_FPS_P119_88, 11988 - 100, 11988 + 100}
+};
+
+mtl_handle shared_st_handle = NULL;
+unsigned int active_session_cnt = 0;
+struct mtl_init_params param = {0};
```

Comment:
Seems these global can be removed? A external global is likely a bug design, if it's really need, pls warp it as function, something like:
```
mtl_handle  mtl_get_handle(args) {
}
void mtl_put_handle(mtl_handle handle) {
}
```

Reply:
Added shared_mtl_handle helper functions mtl_get/set_handle() in kahawai_common.c in https://github.com/OpenVisualCloud/Media-Transport-Library/pull/234/commits/dea527a90227f36a840ef826c3c71b0a0bcd86b6